airbyte-integrations/bases/base-normalization/integration_tests/resources/exchange_rate/catalog.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
how relevant is it that the directory is called exchange_rate? shouldn't we call it with the type of tests that are running?